data/server/king_phisher/server_config.yml

```diff
@@ -107,6 +107,10 @@ server:
   #  some_plugin:
   #    # An option specific to this particular plugin
   #    value_for_some_plugin: true
+  # Plugin Library depenacy path
```

Comment:
These settings should go above `#plugins:` so they're alphabetized. Should go `plugin_directories`, `plugin_install_requirements`, and then `plugin_library_path`.